shuup/core/api/category.py

```diff
@@ -38,6 +43,8 @@ class CategoryFilter(FilterSet):
     shop = django_filters.ModelChoiceFilter(name="shops",
                                             queryset=Shop.objects.all(),
                                             lookup_expr="exact")
+    slug = django_filters.CharFilter(name="translations__slug",
```

Comment:
don't think we should be filtering on slug...can't we filter on identifier instead? or maybe just grab all root categories?